core/CMake_readelf.cmake

```diff
@@ -40,9 +40,16 @@
 # PR 212290: ensure no text relocations (they violate selinux execmod policies)
 # looking for dynamic section tag:
 #   0x00000016 (TEXTREL)                    0x0
+
+include(../drpreloadloc.cmake)
```

Comment:
Hmm, I thought this would take in the file to include, so we don't have to hardcode anything here.

Reply:
Yes we can use ${lib_fileloc}.cmake.